src/Statistics/EmployeeParticipation.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Statistics;
```

Comment:
Bij de andere domain models is er een Models subfolder, daar mag deze class ook in.